specs/http-client/index.html

```diff
@@ -261,19 +266,19 @@ <h2>The <dfn>http:send-request</dfn> function</h2>
 
     <section>
       <h2>Sending a request</h2>
-      <p>The functions defined in this module make one able to send a request to an HTTP server and
-        receive the corresponding response.  Here is how the request is represented by the
-        parameters to this function, and how they are used to generate the actual HTTP request to
-        send.</p>
-
-      <section>
-        <h3>The request elements</h3>
-        <p>The <code>http:request</code> element represents all the needed information to send the
-          HTTP request.  So it is always possible to create such an element that will carry over all
-          the needed info for a particular request.  For some of those values though, you can use an
-          additional param instead.  For instance, some signatures define the
+      <p>The functions defined in this module allow the transmission of a request to an HTTP server, and
```

Comment:
", and" > " and" (remove comma, as what follows is not an independent clause, but belongs to the object of "allow")